src/d3a/d3a_core/area_serializer.py

```diff
@@ -135,8 +135,9 @@ def optional(attr):
             children = [area_from_dict(child) for child in description['children']]
         else:
             children = None
+        transfer_fee_pct = description.get('grid_fee_percentage', None)
```

Comment:
It is no longer a transfer fee, therefore I would recommend to rename it to 'market_fee_pct' or 'area_fee_pct'